templates/eap74-sso-s2i.json

```diff
@@ -46,7 +46,7 @@
             "displayName": "SSO Image Name",
             "description": "Name of the SSO image to use, examples: redhat-sso74-openshift:latest",
```

Comment:
could you please also update the name in the description (or remove it)?

Reply:
@jmesnil Sorry, I must have missed this comment, updated now.